L/libjulia/common.jl

```diff
@@ -88,11 +85,11 @@ function configure(version)
     USE_SYSTEM_PCRE=1
     USE_SYSTEM_OPENLIBM=1
     USE_SYSTEM_DSFMT=1
-    USE_SYSTEM_BLAS=1
-    LIBBLASNAME=libopenblas
-    USE_SYSTEM_LAPACK=1
-    LIBLAPACKNAME=libopenblas
-    # USE_SYSTEM_GMP=1
+    #USE_SYSTEM_BLAS=1
```

Comment:
Is this necessary?

Reply:
Yes. Doesn't make a difference on Linux and fixes macOS

Reply:
Then we should only do this for `1.4.2`, since it did work for `1.5`

Reply:
Actually, I think this will even work with 1.5.x ?

But I've now changed it to restrict this change to macOS, where using `USE_SYSTEM_BLAS=1` has the same issue as on 1.4.x: it does *not* use OpenBLAS, but rather the BLAS shipped with the OS by Apple, period. And that is missing lapack, so it then ends up downloading and building that (even in Julia 1.5); the only reason it works in 1.5 is that the lapack there is newer and more cross compilation friendly. But there is a download in any case. By *not* setting `USE_SYSTEM_BLAS=1`, we get Julia on macOS to use its own BLAS binaries, which works well even on macOS and with Julia 1.4 and 1.5 equally.

Reply:
@vchuravy does that sound plausible / acceptable? I've applied a corresponding change to this PR now.

In any case, I think it would be useful to verify this PR doesn't break Julia 1.5.x builds. I could push a commit which edits the Julia 1.5 builder, to trigger a build of that, and then later drop that commit again once 1.5 builds (again). Alternatively, I could do that in a separate PR. Thoughts?